File: Algoritimos-fundamentais-Entregavel1/Primo.java
import java.util.Scanner;

public class Primo {

    public static void main(String[] args) {

        Scanner teclado = new Scanner(System.in);

        System.out.print("Digite um número: ");
        int numero = teclado.nextInt();

        boolean primo = true;

        if (numero < 2) {
            primo = false;
        } else {

            for (int i = 2; i < numero; i++) {

                if (numero % i == 0) {
                    primo = false;
                    break;
                }
            }
        }

        if (primo) {
            System.out.println("Número Primo!!");
        } else {
            System.out.println("Não é Primo!!");
        }

        teclado.close();
    }
}

File: Algoritimos-fundamentais-Entregavel1/primo.js
const readline = require("readline");

const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

rl.question("Digite um numero: ", (numero) => {

    numero = parseInt(numero);

    let primo = true;

    if (numero <= 1) {
        primo = false;
    }

    for (let i = 2; i < numero; i++) {
        if (numero % i === 0) {
            primo = false;
            break;
        }
    }

    if (primo) {
        console.log("O numero é primo");
    } else {
        console.log("O numero não é primo");
    }

    rl.close();
});